libs/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.html

```diff
@@ -1,6 +1,7 @@
 <div class="chart-container" [style.width]="width" [style.height]="height">
   <div class="chart-container-resize-fix">
     <ng-container *ngIf="datasets && datasets.length > 0; else empty">
+      <div class="error-msg" *ngIf="layerManager.dataRetrievalError">Some data is missing</div>
```

Comment:
Adding an element here might mess up the canvas height because it scales to 100% of its container height. Does the x-axis at the bottom of the chart get cut off when the error message is shown?

Also, let's change the error message to "There was a problem retrieving data. The timeline may be incomplete."

Reply:
Yes, the chart is cut off at the bottom when the message is added. I will update it.

> Adding an element here might mess up the canvas height because it scales to 100% of its container height. Does the x-axis at the bottom of the chart get cut off when the error message is shown?
> 
> Also, let's change the error message to "There was a problem retrieving data. The timeline may be incomplete."